xgcm/grid.py

```diff
@@ -1359,6 +1361,26 @@ def iterate_axis_combinations(items):
         # return the product of the metrics
         return functools.reduce(operator.mul, metric_vars, 1)
 
+    def interp_like(self, da_source, metric):
+        for axname, axis in self.axes.items():
+            try:
+                position_da, _ = axis._get_axis_coord(da_source)
```

Comment:
minor comment: Maybe next we can rename `_get_axis_coord` to `_get_position_name`. Thats literally what the docstring says
> """Return the position and name of the axis coordiante in a DataArray."""